### ISSUE TITLE
Import pwlist4

### ISSUE THREAD

**Issue by aesncast** [aesncast/pw-py#3]
Importing of pwlist4 domains / users / sequences into the local pwlist4 file should be possible.
Example:

```
$ pw-py -i other.pwlist4
importing pwlist4 into local pwlist4
importing domains
importing domain xyz...
importing user abc...
importing sequences
importing sequence foo...
import done
```

**Comment by aesncast** [comment]
Consider collisions in the imported and local files.
Probably add `--force` flag to force overwriting the local entries, otherwise ask user to confirm which entry to use.

**Comment by aesncast** [comment]
Implemented with fef14e42054dc8758dadb81fbe3d30f03d177d8c
